airflow/ui/vite.config.ts

```diff
@@ -19,38 +19,27 @@
 import react from "@vitejs/plugin-react-swc";
 import { defineConfig } from "vitest/config";
 
-// Replace the directory to work with the flask plugin generation
-const transformUrlSrc = () => {
-  return {
-    name: "transform-url-src",
-    order: "post",
```

Comment:
Removed because this property doesn't exist in type `PluginOption`. Add back with a `@ts-expect-error` if needed.